frontend/src/index.scss

```diff
@@ -55,11 +55,6 @@ $standardMargin: 10px;
   cursor: not-allowed;
 }
 
-/* Prevent overlapping with the 🚫 icon */
-.ag-tooltip {
-  transform: translate(15px, -40px);
-}
-
```

Comment:
This is an artifact from an older iteration of the `"This column is read-only"` tooltip and is no longer relevant (no use of the 🚫 icon anywhere)